main.css

```diff
@@ -1,3 +1,31 @@
-h1 {
-    color: red;
+body {
+    font-size: 18px;
```

Comment:
No **magic numbers** in our code.

Reply:
I understand the meaning of magic numbers, However i don't understand why r we calling these magic numbers.